mucommander-viewer-text/build.gradle

```diff
@@ -9,15 +9,20 @@ dependencies {
     api project(':mucommander-encoding')
     api project(":mucommander-preferences")
 
+    comprise group: 'com.fifesoft', name: 'rsyntaxtextarea', version: '3.3.0'
```

Comment:
what's the reason for using 'comprise' here (rather than having an external dependency)?

Reply:
you know, I accept that OSGi exists but I don't like it very :) and this `comprise` thing is still kinda blurry for me (even with your explanation from another issue). I can change to whatever you suggest it will be better :)

Reply:
ack, adding to my todo list to take a further look into that

Reply:
oh that's still on me

Reply:
I'll look at that separately, not reason to block this PR because of that